python/src/lib/workbook_validator.py

```diff
@@ -34,16 +34,16 @@ class WorkbookError:
     col: str
     tab: str
     field_name: str
-    severity: ErrorLevel = ErrorLevel.WARN
+    severity: str = ErrorLevel.WARN.name
```

Comment:
Happy to approve this as is but just curious, why not just implement a serializer method?

Reply:
could you explain in code how that might look like? Happy to implement it! This was the first solution I was able to think of to fix the error I noticed so definitely open to other approaches.

Reply:
This might not be the most elegant solution (I'm not the most up to date on Python best practices), but was thinking something like either subclassing `JSONEncoder` (see [here](https://docs.python.org/3/library/json.html#json.JSONEncoder)), or implementing a custom `toJSON` as here? https://stackoverflow.com/questions/3768895/how-to-make-a-class-json-serializable?page=1&tab=scoredesc#tab-top on `ErrorLevel`. 

That said, I also think this approach is totally fine given it's a v small class / issue :)